docs/modpacks/creating_modpacks.md

```diff
@@ -32,16 +32,23 @@ You can also convert a CurseForge modpack to the Modrinth format via ATLauncher.
 The recommended method to convert packs was previously via packwiz, but we now discourage the use of packwiz for this specific use case. packwiz does not attempt to convert the CurseForge mod versions to Modrinth versions, but ATLauncher does.
 :::
 
-### MultiMC and Prism Launcher
+### MultiMC
 
-MultiMC and Prism are both launchers focused on efficiently managing multiple separate instances. The export instructions for each are identical.
+MultiMC is a launcher focused on efficiently managing multiple separate instances of Minecraft.
 
-You can easily export an instance to the Modrinth format. You just need to select the instance, select the dropdown menu next to Export Instance and click Modrinth. Then, fill out the options and press OK.
+You can easily export a MultiMC instance to the Modrinth format. You just need to select the instance, select the dropdown menu next to Export Instance and click Modrinth. Then, fill out the settings and metadata and press OK.
 
 ![MultiMC export](../../static/img/tutorial/multimcExport.png)
 
+### Prism Launcher
+Prism Launcher is a fork of MultiMC.
+
+Because of this, the export process is fairly similar. You must select the instance, then select the dropdown menu next to Export and click Modrinth. Then, enter the information fields. Optionally, you may check and uncheck files under Files to include or exclude them. Finally, press save and choose the destination.
+
+![Prism Launcher export](../../static/img/tutorial/prismLauncherExport.png)
+
```

Comment:
I don't think it makes sense to split up the sections and then refer one to the other. This isn't quite meant to be read like an article, it's supposed to be a quick reference in order to get people up and running.

Reply:
You can have two screenshots if you want

Reply:
Will be closed if this is not addressed